askama_shared/Cargo.toml

```diff
@@ -10,10 +10,12 @@ workspace = ".."
 
 [features]
 default = []
-serde-json = ["serde", "serde_json"]
+serde-json = []
```

Comment:
So `serde` is now required for deserializing the configuration, but I think `serde_json` should still be optional, as before, no?